docs/data/data-grid/demo/OneMasterDemo.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import {
+  DataGridPremium,
+  GridColDef,
+  GridRenderCellParams,
+  GridToolbarContainer,
+  GridToolbarQuickFilter,
+} from '@mui/x-data-grid-premium';
+import DemoHub, { featuresSet } from './DemoHub';
+import Link from '@mui/material/Link';
+import Chip from '@mui/material/Chip';
+import Avatar from '@mui/material/Avatar';
+import Typography from '@mui/material/Typography';
+
+export const PlanTag = (props: { plan: string }) => {
+  const { plan } = props;
+  function getChipProperties(plan: string) {
+    switch (plan.toLowerCase()) {
+      case 'premium':
+        return { avatarLink: '/static/x/premium.svg', color: '#ffecc8' };
+      case 'pro':
+        return { avatarLink: '/static/x/pro.svg', color: '#c8e9ff' };
+      default:
+        return { avatarLink: undefined, color: '#c8ffdb' };
+    }
+  }
+
+  const chipPropperties = getChipProperties(plan);
+  const avatar = !chipPropperties.avatarLink ? undefined : (
+    <Avatar src={chipPropperties.avatarLink} />
+  );
+  return (
+    <Chip avatar={avatar} sx={{ background: chipPropperties.color }} label={plan} />
+  );
+};
+
+function OneMasterDemo() {
+  const columns: GridColDef[] = [
+    {
+      field: 'name',
+      headerName: 'Feature name',
+      width: 130,
+      renderCell: (params) => {
+        if (!params.value) return;
+        return (
+          <Typography sx={{ fontSize: '1rem', fontWeight: '500' }}>
+            {params.value}
+          </Typography>
+        );
+      },
+    },
+    {
+      field: 'description',
+      headerName: 'Description',
+      flex: 0.5,
+    },
+    {
+      field: 'plan',
+      headerName: 'Plan',
+      width: 130,
+      renderCell: (params: GridRenderCellParams<string>) => {
```

Comment:
```suggestion
      type: 'singleSelect',
      valueOptions: ['Premium', 'Pro', 'Community'],
      renderCell: (params: GridRenderCellParams<string>) => {
```

To improve filtering :)